app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Kindly do well to remove commented code from your project. This keeps your codebase clustered is done repeatedly wich won't be a good practice 👍